website_react/src/components/inputs/InputBoolean.jsx

```diff
@@ -0,0 +1,32 @@
+const InputBoolean = (props) => {
+    return (
+        <div className="form-input-div" key={props.index}>
+            <label className="input-label">{props.formFields.title}</label>
+            <div>
+                <input
+                    type="radio"
+                    id={"true" + props.index}
+                    name={"bool" + props.index}
+                    value="true"
+                    defaultChecked={props.formFields.default_value}
```

Comment:
what happens id default_value is undefined?

Reply:
no field will be selected by default